### DISCUSSION TITLE
[SOLVED] How to get Light mode? Remove empty space?

### DISCUSSION THREAD

**Discussion by Shohreh** [Open-Shell/Open-Shell-Menu#1234] [Q&A] [answered]
Hello,

It's neither a bug nor a feature, but I found no Help section.

I'm not used to OpenShell, and am trying to install it on a new Windows10 host while moving from Windows7.

1. I'd like a Light mode because it's more readable than the default Dark mode. So I looked around, checked the Customize taskbar, and used grey background + black text: As you can see, the text at the bottom is white. How can I get it to use black?
2. There's wasted space between the shortcuts and the running apps: How can I remove it?

Thank you.

![image](https://user-images.githubusercontent.com/1541861/206194498-e7b8d49e-874e-42c9-ab2c-b36fcdfa135a.png)

--
Edit: I found how to remove the empty space — simply unlock the taskbar, and move the section upward —, but it seems impossible to change text from white to black while keeping the Light mode instead of Dark.

**Comment by bonzibudd** [comment, marked as answer]
You should change your Windows color mode to light, so the text and icons will be black.

**Reply by Shohreh** [reply]
Thanks for the tip. I already had it in Light, and couldn't find what else to try in Themes and Colors.

Downloading the latest upgrades solved the issue: Text is now black :-)
